ev3/__init__.py

```diff
@@ -1 +1,5 @@
-__all__ = ['ev3dev', 'lego','mindsensors']
```

Comment:
I've tried to import ev3 in the past, and i must import the other 3 modules along with it.

This fixes that